README.md

```diff
@@ -1,6 +1,8 @@
 # Litmus
 
-**TODO: Add description**
+![Hex.pm](https://img.shields.io/hexpm/v/lazy_agent.svg)
```

Comment:
This is the badge for `lazy_agent`, we might want to wait until we have this up on Hex before adding our badge.